Change take_screenshots password and add user

sofiasyria opened the pull request:
After the agama_tumbleweed test decided to run the agama_base post_run_hook, it fails because of wrong root password and missing used "bernhard". This PR attempts to fix this issue
See failure: https://openqa.opensuse.org/tests/3458170#step/agama_installation/15
VR: https://openqa.opensuse.org/tests/3482079

Edited YUPDATE_GIT: sofiasyria/e2e-agama-playwright#too_fast in the openqa job group for now, as Joaquin who can merge is on vacation. This needs to be changed back to YUPDATE_GIT: jknphy/e2e-agama-playwright#main once merged.

lemon-suse commented:
It seems you only update the user and password, but update the indent will cause the version of take_screenshots far from the origin one. I'm not sure how to deal with it, or we can rename it as a new ts?

sofiasyria commented:
> It seems you only update the user and password, but update the indent will cause the version of take_screenshots far from the origin one. I'm not sure how to deal with it, or we can rename it as a new ts?

I don't understand. How changing the indentation is changing the test itself? If you have a hard time identifying the actual changes I made to the code, it's the root password and I added some lines to add user bernhard and set password, just so that the post_run_hook of agama_base will pass. Particularly lines {72..76} are new. Would it help if I split the commits?

lemon-suse commented:
> > It seems you only update the user and password, but update the indent will cause the version of take_screenshots far from the origin one. I'm not sure how to deal with it, or we can rename it as a new ts?
> 
> I don't understand. How changing the indentation is changing the test itself? If you have a hard time identifying the actual changes I made to the code, it's the root password from linux to nots3cr3t and I added some lines to add user bernhard and set password, just so that the post_run_hook of agama_base will pass. Particularly lines {72..76} are new. Would it help if I split the commits?

I just mean we'd better do not modify this version of take_screenshots too much, especially unneeded ident changed. Because I'm not sure how to deal with the difference between it and the original one in parent repo. So maybe the good way is to create a new test case copied from take_screenshots. This is just discussion.

sofiasyria commented:
> > > It seems you only update the user and password, but update the indent will cause the version of take_screenshots far from the origin one. I'm not sure how to deal with it, or we can rename it as a new ts?
> > 
> > 
> > I don't understand. How changing the indentation is changing the test itself? If you have a hard time identifying the actual changes I made to the code, it's the root password from linux to nots3cr3t and I added some lines to add user bernhard and set password, just so that the post_run_hook of agama_base will pass. Particularly lines {72..76} are new. Would it help if I split the commits?
> 
> I just mean we'd better do not modify this version of take_screenshots too much, especially unneeded ident changed. Because I'm not sure how to deal with the difference between it and the original one in parent repo. So maybe the good way is to create a new test case copied from take_screenshots. This is just discussion.

I left the indentation changes out of this PR. I don't see a point to make a new test and keep old files that are not used anywhere. Git is great in keeping history and we can always revert a change in the future.

jknphy commented:
As discussed, let's create new ticket to stop using take_screenshots and have a default installation for Agama, avoiding sync work as much as possible of ci tests like this one with upstream project Agama. Please paste ticket here for reference and perhaps we can close the PR.
